pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1720,15 +1720,17 @@ func publishDatastoreConfig(ctx *getconfigContext,
 }
 
 func parseVolumeRefList(volumeRefConfigList []types.VolumeRefConfig,
-	volumeRefs []*zconfig.VolumeRef) {
+	volumeRefs []*zconfig.VolumeRef, appUUID uuid.UUID) {
 
 	var idx int
 	for _, volumeRef := range volumeRefs {
 		volume := new(types.VolumeRefConfig)
 		volume.VolumeID, _ = uuid.FromString(volumeRef.Uuid)
 		volume.GenerationCounter = volumeRef.GenerationCount
-		volume.RefCount = 1
+		volume.AppUUID = appUUID
 		volume.MountDir = volumeRef.GetMountDir()
```

Comment:
Didn't you remove the MountDir? Or only from VolumeRefStatus?

Reply:
`volume` in this case is of type `VolumeRefConfig`, so MountDir here makes sense. however the name of the variable is confusing, so I renamed it to `vrc` now